monopod/pkg/commands/readme.go

```diff
@@ -1,19 +1,43 @@
 package commands
 
 import (
+	"embed"
+	_ "embed"
 	"fmt"
-	"os"
-	"path"
 	"sort"
-	"strings"
+	"text/template"
 
 	"github.com/spf13/cobra"
 
 	"github.com/chainguard-images/images/monopod/pkg/commands/options"
-	"github.com/chainguard-images/images/monopod/pkg/constants"
 	"github.com/chainguard-images/images/monopod/pkg/images"
 )
 
+type completeReadme struct {
+	ReadmeFile     string `tfsdk:"readme_file" hcl:"readme_file"`
+	ShortDesc      string `tfsdk:"short_description" hcl:"short_description"`
+	ConsoleSummary string `tfsdk:"console_summary" hcl:"console_summary"`
+	Image          string `tfsdk:"image" hcl:"image"`
+	Name           string `tfsdk:"name" hcl:"name"`
+	Logo           string `tfsdk:"logo" hcl:"logo"`
+	EndOfLife      string `tfsdk:"endoflife" hcl:"endoflife"`
```

Comment:
Not sure if this PR is the best place for this comment - but i've some reservations about including EOL in our image docs.

We might wrongly imply that an image is not EOL if we forget / miss to update the docs. Also it can be difficult to determine this, some add a README note, others may publish it on a website etc.

Perhaps if we only ever include an EOL reference when one is specifically added? otherwise we don't include any reference or placeholder to anything EOL in the docs at all?

Thoughts?

Reply:
It is a good question, glad you called this out @mamccorm. Right now that field is not used in the rendered README.md files. However, I've designed it in so that if/when showing EOL data becomes a requirement, or these files become a source of truth for other systems (like an API), the field is there and we can use it.